components/BankPayouts/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import Image from 'next/image'
+import HandWithPenny from 'public/images/handWithPenny.png'
+import LzButton from 'components/UI/button'
+import { FaAngleRight } from 'react-icons/fa'
+import { periphery } from './data'
+import LzInput from 'components/UI/input'
+
+interface MyProps {
+  title: string
+  desc: string
+  label: string
+  type: string
+  id: string
+}
+
+export const PayoutLHS = ({ title, desc, label, id, type }: MyProps) => {
```

Comment:
If there wont be any special logic in this component, just return the component directly. Eg

```jsx
const MyComponent = ()=> (
<div>Content</div>
)
```

Reply:
Sorted!